traits/adaptation/adaptation_manager.py

```diff
@@ -16,7 +16,7 @@
 from heapq import heappop, heappush
 import inspect
 import itertools
-import sys
+import six
```

Comment:
Import order: `import six` should be in a separate group after the standard library imports.